app/pages/site.py

```diff
@@ -229,39 +244,67 @@
         'bottomMargin': 5,
     },
     children=[
-        # Move the logo outside of dbc.Container
+        # HEADER
+        html.Div(id='gc-header-container'),
         html.Div(
+            children=gc_line(
+                border_width=3,
+                line_width=5,
+                color='red',
+                margin='0 0 10px 20px'
+            ),
```

Comment:
Is this red line from the GC look and feel style components? What's it's purpose?

Reply:
Red line is similar to what is found in the header of many GC websites like [this one](https://design.canada.ca/common-design-patterns/signature.html). We can adjust styling like width or color further if needed.